lib/engine/game/g_18_norway/steps/dividend.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require_relative '../../../step/dividend'
+require_relative '../../../step/half_pay'
+require_relative '../../../step/minor_half_pay'
```

Comment:
Are there minor companies in 18Norway?

Reply:
No there is not any minors in 18Norway, did copy a bit to much from 1846
Now removed